ytfzf

```diff
@@ -995,13 +1018,15 @@ parse_opt () {
 			# Not to be used explicitly
 			;;
 
+		N)	YTFZF_NOTI=${optarg:-1} ;;
+
 		*)
 			usageinfo
 			exit 2 ;;
 	esac
 }
 
-while getopts "LhDmdfxHaArltSsvn:U:-:" OPT; do
+while getopts "LhDmdfxHaArltSsvnN:U:-:" OPT; do
```

Comment:
here option `n` also takes a `OPTARG`, it should be `n:`

Reply:
Thanks for the tips, was a little confuse at that, now flag -N now is work without arg i also change the -h description for it